iocore/net/I_UDPPacket.h

```diff
@@ -63,48 +63,9 @@ class UDPPacket
   int from_size;
 
   LINK(UDPPacket, link);
-};
-
-/**
-   Create a new packet to be sent over UDPConnection. This actually
-   copies data from a buffer.
 
-
-   @param to  address of where to send packet
-   @param when ink_hrtime relative to ink_get_hrtime_internal()
-   @param buf if !nullptr, then len bytes copied from buf and made into packet.
-   @param len # of bytes to copy from buf
- */
```

Comment:
Looks like these docs are being dropped in the diff.

Reply:
Yeh, again my fault, and honestly I don't find those useful anyways. But yeh, we can ask Chris to restore them. One thing though, several of those "extern" declarations had no implementation anywhere, so they were there, documented, and not usable.